diff --git a/src/manage.c b/src/manage.c
index b0684d6b..873dc5c3 100644
--- a/src/manage.c
+++ b/src/manage.c
@@ -467,6 +467,7 @@ void manage_window(xcb_window_t window, xcb_get_window_attributes_cookie_t cooki
         xcb_reply_contains_atom(type_reply, A__NET_WM_WINDOW_TYPE_UTILITY) ||
         xcb_reply_contains_atom(type_reply, A__NET_WM_WINDOW_TYPE_TOOLBAR) ||
         xcb_reply_contains_atom(type_reply, A__NET_WM_WINDOW_TYPE_SPLASH) ||
+        xcb_reply_contains_atom(type_reply, A__NET_WM_WINDOW_TYPE_NOTIFICATION) ||
         xcb_reply_contains_atom(state_reply, A__NET_WM_STATE_MODAL) ||
         (cwindow->max_width > 0 && cwindow->max_height > 0 &&
          cwindow->min_height == cwindow->max_height &&
